script/Ciel/Day 10/10_01_CIEL10_6.txt

```diff
@@ -1272,22 +1408,23 @@ Ciel deflects them with a swing of her right arm.
 [sha:13049cc3a92e08e728348513c05d8f5f11bb478f]{
 -- Page 506, Offset 28491.
 -- 実力差がある事を<女|ノエル>は承知している。
-The woman is aware of the gap in strength between them.
+The woman knows that she can't measure up to Ciel in terms of strength.
 }
 [sha:8450ab9187821df1b46459ea69cf344b50ad311e]{
 -- Page 506, Offset 28492.
 -- 左手を限界まで酷使して放った黒鍵三本は<囮|おとり>にすぎない。
-The three Black Keys that she released by pushing her left hand to its maximum are nothing but decoy.
+The three Black Keys she launched by pushing her left hand to its very limit are nothing more than diversion.
```

Comment:
"a diversion?" Is diversion uncountable?

Reply:
Nope just a typo